lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -1072,7 +1078,8 @@ def test_coefficients_truncated_gaussian_default_initial_guess(self):
         expected values, and the coefficient names also match
         expected values for a truncated Gaussian distribution, where the
         default values for the initial guess are used, rather than using a
-        linear least-squares regression to construct an initial guess.."""
+        linear least-squares regression to construct an initial guess."""
```

Comment:
Can you expand this comment to point out that in the limit of a finer tolerance this result will match the case where a linear least-squares regression has been used to construct the initial guess.